sim/config.json

```diff
@@ -3,8 +3,8 @@
     "robot": {
       "w": 1.968504,
       "h": 1.968504,
-      "starting_x": 30,
-      "starting_y": 40,
+      "starting_x": 5,
+      "starting_y": 5,
       "starting_angle": 0
     },
     "field": {
```

Comment:
Whilst you're here, you probably want to remove this — update your pyfrc installation if you haven't seen it complain yet.

There's also a new thing in pyfrc where you can select the starting position from a list. We should definitely use that too.